### ISSUE TITLE
Not working for me

### ISSUE THREAD

**Issue by lamba** [Waterflames/AutoRefresh#11]
I enabled the package and auto refresh for the file using command palette. But the file isn't getting stored in the settings and I don't see the file auto refreshing either.

**Comment by Wouterdek** [comment]
I just tested the plugin on version 3.2.2 of sublime and it seems to work fine here. 
Under the View menu, select View Console and see if there are any relevant errors.

**Comment by lamba** [comment]
The only useful message I see on the console is the following when I go to Preferences > Package Settings > AutoRefresh > Settings - Default & User

`Unable to open /Users/lamba/Library/Application Support/Sublime Text 3/Packages/Auto Refresh/AutoRefresh.sublime-settings
`

The plugin doesn't seem to be writing the settings out to the settings file when I enable the package and run the "Remember" for this file command. I don't see the settings file when I check the folder. 

I did create user-level settings and those got saved, but it doesn't seem to be helping to auto-refresh the log file I'm working with. The file only refreshes if I tab away and tab back. If I set the following, it works but I don't want the annoying dialog popping up every 2 seconds. 

`"always_prompt_for_file_reload": false`

My Sublime version is 3.2.2 build 3211

**Comment by Wouterdek** [comment]
Are you making changes to the file in sublime? Because if you are, and you disable `always_prompt_for_file_reload`, then sublime will not reload the file.

> The file only refreshes if I tab away and tab back

That is built-in behavior of Sublime.

**Comment by lamba** [comment]
No, it's a log file that is constantly being updated by another app and I want it to reload in real-time in Sublime, if possible so I can monitor for certain regex patterns etc.

**Comment by Wouterdek** [comment]
The bug is not in the settings. The `Unable to open` message occurs because the settings file doesn't exist, which is normal if you haven't changed any settings.
I've tried, but I can't reproduce this bug, which makes it basically impossible to see where things are going wrong. I'll leave this issue open in case someone else has the same problem.

**Comment by Salil999** [comment]
I had to manually create the files with the default settings for it to work.
